cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -29,6 +29,7 @@
 import androidx.core.content.ContextCompat;
 
 import com.cleverpush.banner.AppBannerModule;
+import com.cleverpush.banner.WebViewActivity;
```

Comment:
Why does the WebViewActivity live under `banner ` package? In this case we use it for banners + for notifcations.
Please move it to somewhere else.